osf/exceptions.py

```diff
@@ -179,9 +179,15 @@ class RegistrationBulkCreationRowError(OSFError):
     """Raised if a draft registration failed creation during bulk upload"""
 
     def __init__(self, upload_id, row_id, title, external_id, draft_id=None, error=None, approval_failure=False):
-        self.short_message = 'Title: {}, External ID: {}'.format(title, external_id)
-        self.long_message = 'Draft registration creation failed: [upload_id={}, row_id={}, ' \
-                            'title={}, external_id={}]'.format(upload_id, row_id, title, external_id)
+
+        # `draft_id` is provided when the draft is created but not related to the row object
         self.draft_id = draft_id
-        self.error = error if error else 'Draft registration creation error'
+        # `approval_failure` determines whether the error happens during the approval process
         self.approval_failure = approval_failure
+        # The error information for logging, sentry and email
+        self.error = error if error else 'Draft registration creation error'
```

Comment:
How about we make `error='Draft registration creation error'` by default in the parameter?

Reply:
As discussed, will leave as it is to avoid super long function signatures.